_features/lin_advance.md

```diff
@@ -19,11 +19,43 @@ Since the root cause is pressure, `LIN_ADVANCE` de-couples extrusion from the ot
 ## Advantages
 
 - Better dimensional precision due to reduced bleeding edges.
-- Higher printing speeds are possible without any loss of print quality.
+- Higher printing speeds are possible without any loss of print quality - as long as your extruder can handle the needed speed changes.
 - Visible and tangible print quality is increased even at lower printing speeds.
 - No need for high acceleration and jerk values to get sharp edges.
 
 <br>
+
+## Special notes for v1.5
+
+### Changelog
+
+- K is now a meaningful value with the unit [mm of filament compression needed per 1mm/s extrusion speed] or [mm/mm/s].
+- Load inside stepper ISR reduced as no calculations are needed there any more. Instead, the extruder runs at a fixed speed offset during pressure adjustment. Therefore this version runs faster.
+- LIN_ADVANCE now respects hardware limitations set in Configuration.h, namely extruder jerk. If the pressure corrections would need faster adjusments than allowed by extruder jerk limit, the acceleration for this print move is limited to a value which allowes to use extruder jerk speed as the upper limit.
```

Comment:
"need faster **adjusments** than allowed" --> "need faster **adjustments** than allowed"

"limited to a value which **allowes** to" --> "limited to a value which **allows** to"